outputs.tf

```diff
@@ -24,6 +24,12 @@ output "namespace" {
   depends_on  = [null_resource.argocd-config]
 }
 
+output "service_account" {
+  description = "The name of the service account for the ArgoCD instance has been provisioned"
+  value       = local.service_account_name
```

Comment:
@seansund isn't this suppose to be `local.argocd_config_values.service_account_name` ?